### PR TITLE
XAMARIN-33322-Update the chart datamarker label ug document

### PR DESCRIPTION
**Pull request by YuvarajPalanisamy**
Updated the code example, when we set LabelTemplate property need to set SegmentLabelContent property becomes LabelContentPath.

For more details, please refer the below link
https://support.syncfusion.com/agent/tickets/363768

### CODE REVIEW

File: wpf/Charts/Adornments/Label.md [hunk 1]
```diff
@@ -100,7 +100,7 @@ The following code example demonstrates the customization of label using the abo
 
 ## Label Template
 
-The default appearance of the label can be customized using [`LabelTemplate`](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Charts.ChartAdornmentInfoBase.html#Syncfusion_UI_Xaml_Charts_ChartAdornmentInfoBase_LabelTemplate) property as in the below code example:
+The default appearance of the label can be customized using [`LabelTemplate`](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Charts.ChartAdornmentInfoBase.html#Syncfusion_UI_Xaml_Charts_ChartAdornmentInfoBase_LabelTemplate) property and along with [`SegmentLabelContent`](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Charts.ChartAdornmentInfoBase.html#Syncfusion_UI_Xaml_Charts_ChartAdornmentInfoBase_SegmentLabelContent) is LabelContentPath as in the below code example:
```
Comment (Manivannan-E) [inline wpf/Charts/Adornments/Label.md:103]: @YuvarajPalanisamy Add the additional information content in SegmentLabelContent explaination table for LabelContentPath.

Comment (YuvarajPalanisamy) [reply wpf/Charts/Adornments/Label.md:103]: @Manivannan-E  modified the description for LabelContentPath value of SegmentLabelContent property.

**Review by Manivannan-E — APPROVED** [review]
Changes are fine.

**Review by SyncfusionKarthikeyan — APPROVED** [review]
This is fine

**Pull request merged**